server/channels/app/limits.go

```diff
@@ -38,10 +23,17 @@ func (a *App) GetServerLimits() (*model.ServerLimits, *model.AppError) {
 		limits.MaxUsersLimit = maxUsersLimit
 		limits.MaxUsersHardLimit = maxUsersHardLimit
 	} else if license != nil && license.IsSeatCountEnforced && license.Features != nil && license.Features.Users != nil {
-		// Enforce license limits as required by the license with grace period.
+		// Enforce license limits as required by the license with configurable extra users.
```

Comment:
nit: I know I'm late to the conversation on this but without this previous context on how we got here, extra users is a bit confusion on what are these extra users for. We should keep in the code some reference that this are for "grace period" 
```suggestion
		// Enforce license limits as required by the license with configurable extra grace users.
```